src/main/java/com/google/edith/GroceryDataReader.java

```diff
@@ -0,0 +1,105 @@
+package com.google.edith;
+
+import com.opencsv.CSVReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+/** 
+  * Processes data file of product prices and sizes at different
+  * stores and returns the best value item. 
+  */
+public final class GroceryDataReader {
+
+    private static final String ALDI = "Aldi";
+    private static final String KROGER = "Kroger";
+    private static final String TRADERJOES = "Trader Joe's";
```

Comment:
nit: For these vars we should use UPPER_SNAKE_CASE so TRADERJOES should be TRADER_JOES. I.e. add a _ where a space should be